config/openshift/v4_20_exp/translate.go

```diff
@@ -12,14 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.)
 
-package v4_19_exp
+package v4_20_exp
 
 import (
 	"net/url"
 	"strings"
 
 	"github.com/coreos/butane/config/common"
-	"github.com/coreos/butane/config/openshift/v4_19_exp/result"
+	"github.com/coreos/butane/config/openshift/v4_20_exp/result"
 	cutil "github.com/coreos/butane/config/util"
 	"github.com/coreos/butane/translate"
```

Comment:
s/ToMachineConfig4_19Unvalidated/ToMachineConfig4_20Unvalidated/ later in this file.

A comment needs an update as well.

Reply:
ToMachineConfig4_19Unvalidated + comment as well need an update

Reply:
ToConfigBytes 4.17 -> 4.20